### ISSUE TITLE
[SUGGESTION]  be able to buy all butation berry

### ISSUE THREAD

**Issue by CometWing** [pokeclicker/pokeclicker#2215]
add a shhop to buy matation berry for peoble who arn't good for it

**Comment by NiCeDiCe90** [comment]
What's the point of the farm if you can just buy them? Makes no sense to me.
Some berries can already be found in dungeons. That should be helpful for people who don't like the farm.

**Comment by Katsunee** [comment]
yes berries mutation berry are just so hard to get. 3  i'l try to get a Lum since 3 days , we realy need a shop to get these berries (maybe to kalos or unova)

**Comment by Jaaslet** [comment]
No one is forcing you to use the farming minigame. I don't want to use it, and it have not stopped me from enjoying the rest of the game.
Adding this shop just removed the use of an optional minigame, for no reason

**Comment by amative1** [comment]
Berries/Farming is intentionally hard, and is entirely optional. The only berries that have been (and currently will be) added to shops are those requiring Oak Items, and only for those who are doing the No Oak Items challenge.
